### PR TITLE
Changed struct param names as preferred

### PR DESCRIPTION
**Pull request by AngithCJ**
**Changes**
- Changed struct param names as preferred by go lint

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -7221,7 +7221,7 @@ curl -i GET \
         "HealthRollup": "OK"
     },
     "ServiceEnabled": true,
-    "HTTPPushURI": "",
```
Comment (jeevan-kamkar) [inline docs/README.md:7224]: remove the doc update

File: svc-update/README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ curl -i GET \
       "HealthRollup":"OK"
    },
    "ServiceEnabled":true,
-   "HTTPPushURI":"",
```
Comment (jeevan-kamkar) [inline svc-update/README.md:62]: remove the doc update

File: svc-update/uresponse/updateservice.go [hunk 3]
```diff
@@ -76,16 +76,16 @@ type UpdateService struct {
 	response.Response
 	Status                        Status                    `json:"Status"`
 	ServiceEnabled                bool                      `json:"ServiceEnabled"`
-	HTTPPushURI                   string                    `json:"HTTPPushURI"`
+	HTTPPushURI                   string                    `json:"HttpPushUri"`
 	FirmwareInventory             FirmwareInventory         `json:"FirmwareInventory"`
```
Comment (jeevan-kamkar) [inline svc-update/uresponse/updateservice.go:80]: json tag update in not required

File: svc-update/uresponse/updateservice.go [hunk 4]
```diff
@@ -76,16 +76,16 @@ type UpdateService struct {
...
 	SoftwareInventory             SoftwareInventory         `json:"SoftwareInventory"`
 	Actions                       Actions                   `json:"Actions"`
 	OEM                           *OEM                      `json:"Oem,omitempty"`
 	ClientCertificates            *ClientCertificates       `json:"ClientCertificates,omitempty"`
-	HTTPPushURIOptions            *HTTPPushURIOptions       `json:"HTTPPushURIOptions,omitempty"`
-	HTTPPushURIiOptionsBusy       bool                      `json:"HTTPPushURIiOptionsBusy,omitempty"`
-	HTTPPushURITargets            []string                  `json:"HTTPPushURITargets,omitempty"`
-	HTTPPushURITargetsBusy        bool                      `json:"HTTPPushURITargetsBusy,omitempty"`
```
Comment (jeevan-kamkar) [inline svc-update/uresponse/updateservice.go:88]: check for the json tag update

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested change

File: docs/README.md (hunk 1 above)
Comment (AngithCJ) [reply docs/README.md:7224]: I changed HttpPushUri to HTTPPushURI in this [PR](https://github.com/AngithCJ/ODIM/pull/3). I am removing that changes here.

File: svc-update/README.md (hunk 2 above)
Comment (AngithCJ) [reply svc-update/README.md:62]: same as above

File: svc-update/uresponse/updateservice.go (hunk 3 above)
Comment (AngithCJ) [reply svc-update/uresponse/updateservice.go:80]: removing the tag update which was done in earlier PR

File: svc-update/uresponse/updateservice.go (hunk 4 above)
Comment (AngithCJ) [reply svc-update/uresponse/updateservice.go:88]: same as above

**Review by amar-shalgar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Pull request merged**